cfme/infrastructure/provider/__init__.py

```diff
@@ -285,6 +283,15 @@ def hosts(self):
             result.append(Host(name=host["name"], credentials=cred))
         return result
 
+    def get_clusters(self):
+        """returns the list of clusters belonging to the provider"""
+        web_clusters = []
+        sel.force_navigate('infrastructure_clusters')
```

Comment:
My error, forgot to change this when moved this block from Clusters. Review is really cool thing !